src/main/java/com/gfl/client/model/dto/ProxyConfigHolder.java

```diff
@@ -0,0 +1,14 @@
+package com.gfl.client.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
```

Comment:
`@Data` already contains ` @RequiredArgsConstructor`
You can remove `@NoArgsConstructor, @AllArgsConstructor`